src/types.rs

```diff
@@ -115,7 +121,13 @@ impl Network {
 	/// Returns `true` if the network is a Polkadot _parachain_.
 	pub(super) fn is_polkadot_para(&self) -> bool {
 		use Network::*;
-		matches!(self, PolkadotAssetHub | PolkadotCollectives | PolkadotBridgeHub | PolkadotPeople)
+		matches!(
+			self,
+			PolkadotAssetHub
+				| PolkadotCollectives
+				| PolkadotBridgeHub
+				| PolkadotPeople | PolkadotCoretime
```

Comment:
Funny formatting but `lint` allows it

Reply:
Yea the lint actually enforced it, I originally wrote them on multiple lines and the linter combined these on the same line 🧐